Components/Alerts/alert.css

```diff
@@ -0,0 +1,64 @@
+
+.alert {
+  box-sizing: border-box;
+  width: 100%;
+  border-radius: 5px;
+  padding: 1.25rem 2.5rem;
+  box-shadow: 0 0 15px 5px #ccc;
+  border-left: 0.5rem solid;
```

Comment:
using rem and px , and instead of 0.5rem , it could have been in px??

Reply:
Thanks for bringing it to my attention. Making the changes